changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 31st August 2023
+
+* Extended [Get all reservations](../operations/reservations.md#Get-all-reservations-ver-2023-06-06) response with `ScheduledStartUtc` and `ActualStartUtc`.
```

Comment:
```suggestion
 * Extended [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06) response with `ScheduledStartUtc` and `ActualStartUtc`.
```

Reply:
Thanks! Though the leading space shouldn't be there, no?

Reply:
Yeah wrong line 🙈 there is a space in the next line 6 that should not be there.

Reply:
Or is that intended?

Reply:
Yeah, the next list item is a subitem of this one, so that was intentional.